lintipy.py

```diff
@@ -21,6 +21,14 @@
 PULL_REQUEST_EVENT = 'pull_request'
 
 
+ERROR = 'error'
+FAILURE = 'failure'
+PENDING = 'pending'
+SUCCESS = 'success'
+
+STATUS_STATES = {ERROR, FAILURE, PENDING, SUCCESS}
```

Comment:
What do you think if we add a link to a documentation for these?
I did not see any usage of `ERROR` in this PR, so I instantly started digging into a place where this was defined.
So, maybe a link would help: https://developer.github.com/v3/repos/statuses/

Reply:
Very good points, the error is actually what I would suggest as a state for timeouts.